src/types/account_info_type.py

```diff
@@ -0,0 +1,12 @@
+from graphene import List, String, ObjectType
+
+class TransactionType(ObjectType):
+  name = String(required=True)
+  timestamp = String(required=True)
+
+class AccountInfoType(ObjectType):
+  brbs = String(required=True)
+  cityBucks = String(required=True)
```

Comment:
`city_bucks`

Reply:
😠 i didn't write this, but i'll make the change